src/libImaging/Histo.c

```diff
@@ -75,6 +84,9 @@ ImagingGetHistogram(Imaging im, Imaging imMask, void* minmax)
     }
 
     h = ImagingHistogramNew(im);
+    if (h == NULL) {
```

Comment:
```suggestion
    if (!h) {
```
Suggestion. Looking through the rest of the Pillow C code, this is just the more consistent form.